### PR TITLE
Add slots docs & fix "Animation" docs typo

### LINKED ISSUE #748: Documentation typo & missing documentation on slots
_Note: no demo/repro. steps because this is purely a documentation issue._

- [Animations docs page](https://googlewebcomponents.github.io/model-viewer/examples/animation.html) has a missing space typo: "theautoplay"
- Missing documentation on [slots](https://github.com/GoogleWebComponents/model-viewer/blob/master/src/template.ts#L240-L284)

Will submit a PR soon addressing these issues.

### PR DESCRIPTION
**Pull request by cwervo**
- Fixes missing space in Animation docs page
- Adds beginning of documentation about using slots in &lt;model-viewer&gt;

### Reference Issue

Fixes #748

### CODE REVIEW

**Comment by cdata** [comment]
@cwervo if you rebase this change you should pick up a fixed Travis config that will help the builds go green again.

File: index.html [hunk 1]
```diff
@@ -82,6 +82,8 @@ <h3 class="grouping-title border-bottom">Examples</h3>
             <li><a href="examples/lighting-and-environment.html"><h4>Lighting &amp; Environment</h4></a></li>
 
             <li><a href="examples/augmented-reality.html"><h4>Augmented Reality</h4></a></li>
+
+            <li><a href="examples/overriding-slots.html"><h4>Overriding Slots</h4></a></li>
```
Comment (cdata) [inline index.html]: Can we change this to "Customizing UI"?

File: examples/overriding-slots.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>&lt;model-viewer&gt; Animation</title>
```
Comment (cdata) [inline examples/overriding-slots.html]: Change "Animation" to "Customizing UI"

**Comment by cwervo** [comment]
@cdata Made the images 5%KB of what they used to be & renamed to "Customizing UI" 😄

**Pull request merged**